ui/src/main/java/edu/wpi/grip/ui/PaletteController.java

```diff
@@ -24,30 +18,24 @@
 
 /**
  * Controller for a list of the available operations that the user may select from
- *
- * @see OperationAddedEvent
  */
 @Singleton
 public class PaletteController {
 
-    @FXML
-    private VBox root;
-    @FXML
-    private VBox operations;
-    @FXML
-    private CustomTextField operationSearch;
-    @Inject
-    private OperationController.Factory operationControllerFactory;
-    @Inject
-    private EventBus eventBus;
-    @Inject
-    private Palette palette;
-    private ControllerMap<OperationController, Node> operationsMapManager;
+    @FXML private VBox root;
+    @FXML private CustomTextField operationSearch;
+    @FXML private Tab allOperations;
+    @FXML private Tab imgprocOperations;
+    @FXML private Tab featureOperations;
+    @FXML private Tab networkOperations;
+    @FXML private Tab opencvOperations;
+    @FXML private Tab miscellaneousOperations;
```

Comment:
What do you mean commit noise?  I literally have to inject these so I can [set some properties on them](https://github.com/WPIRoboticsProjects/GRIP/pull/502/files#diff-e60dcd0c2e9fa0f698d98ffeda803301R43).